Dev4Agriculture.ISO11783.ISOXML/GeometryUtility.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Dev4Agriculture.ISO11783.ISOXML.TaskFile;
+
+namespace Dev4Agriculture.ISO11783.ISOXML
+{
+    public static class GeometryUtility
+    {
+        private const decimal EquityTolerance = 0.000000001m;
+
+        private static List<ISOPoint> Order(this List<ISOPoint> points)
```

Comment:
The name of the function is not complete, I would estimate that it's something like OrderPointsByDistanceFromCenter?